label_studio/core/utils/common.py

```diff
@@ -148,16 +148,22 @@ def paginator(objects, request, default_page=1, default_size=50):
 
```

Comment:
why does this method contain a `DEPRECATED` and a TODO to use djp paginator? looks like it does use it. 
can the deprecation text be removed?

Reply:
Comment about django rest framework pagination. 
https://www.django-rest-framework.org/api-guide/pagination/

We already use it: https://github.com/heartexlabs/label-studio/blob/master/label_studio/projects/api.py#L88
and I think we should use it for all cases.

Reply:
My point was the deprecated comment was confusing since this method does already use Django pagination.

I get it, you feel this code can be removed and replaced with the new code.

ok, for now, I am just making this change. I will leave the TODO in place for a code refactor at another time